crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -106,6 +106,39 @@ fn removed_name(checker: &mut Checker, expr: &Expr, ranged: impl Ranged) {
                 ["airflow", "utils", "dates", "date_range"] => {
                     Some((qualname.to_string(), Replacement::None))
                 }
+                ["airflow", "triggers", "external_task", "TaskStateTrigger"] => {
+                    Some((qualname.to_string(), Replacement::None))
+                }
+                ["airflow", "utils", "file", "TemporaryDirectory"] => {
+                    Some((qualname.to_string(), Replacement::None))
+                }
+                ["airflow", "utils", "file", "mkdirs"] => {
+                    Some((qualname.to_string(), Replacement::None))
+                }
```

Comment:
These have replacements (the stdlib `tempfile`), they should be mentioned.

Reply:
sure! will do

Reply:
this is addressed